src/main/resources/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildTrigger/config.jelly

```diff
@@ -31,6 +31,9 @@
     <f:entry title="Build only if PR is mergeable" field="checkMergeable">
       <f:checkbox default="false"/>
     </f:entry>
+    <f:entry title="Probe the Stash REST API for merge status just so it recalculates the refspec" field="checkProbeMergeStatus">
+      <f:checkbox default="true"/>
```

Comment:
Personally I'm using job-dsl anyway, so it's not affecting me.

Still I think we should make the default false just to keep the previous behavior for sake of least-astonishment.

Reply:
Not sure... it is an intrusive operation a bit (causing load on Stash), but if you poll this repo from a job you also want to be certain to catch new commits using up to date refspecs and in general look at a consistent data view.

Reply:
As discussed in the other thread this might not be generally useful, only for some specific configurations or stash versions. That would be another argument for making it disabled by default

Reply:
OK, you convinced me :)